.storybook/theme.js

```diff
@@ -0,0 +1,36 @@
+import { create } from '@storybook/theming/create';
+import { colorMap, FORM_ELEMENT_BORDER } from '../src/components/dev-hub/theme';
+
+export default create({
+    base: colorMap.pageBackground,
+    colorSecondary: colorMap.teal,
+
+    // UI
+    appBg: colorMap.pageBackground,
+    appContentBg: colorMap.devBlack,
+    appBorderColor: colorMap.greyLightOne,
+    appBorderRadius: 4,
+
+    // Typography
+    fontBase: `akzidenz, -apple-system, BlinkMacSystemFont, "Segoe UI",
+            Helvetica, Arial, sans-serif, 'Apple Color Emoji', "Segoe UI Emoji",
+            "Segoe UI Symbol"`,
+
+    // Text colors
+    textColor: '#fff',
+    textInverseColor: colorMap.devWhite,
```

Comment:
How does this work? These two values are both white, what makes one 'inverse'? I think we should avoid #fff if possible since it's not part of the theme.